docs/metrics.md

```diff
@@ -0,0 +1,113 @@
+# METRICS
+
+## Data Analysis
+
+The collected data will primarily be used to answer the following questions.
+Images are used for visualization and are not composed of actual data.
+
+### Do people use this?
+
+What is the overall engagement of Snooze Tabs? **This is the standard Daily
+Active User (DAU) and Monthly Active User (MAU) analysis.** This captures data
+from the people who have the add-on installed, regardless of whether they are
+actively interacting with it.
+
+### Immediate Questions
+
+TBD
```

Comment:
This is from the Product Vision:
> How will we know we are successful?  Key metric here will be retained users. At what rate do experimenters snooze their tabs? Do users snooze tabs with some consistency or does the novelty wear of quickly?
but they could probably use some word tweaking.  /cc @johngruen @mattgrimes @sharonbautista